bank_accounts.py

```diff
@@ -10,21 +20,83 @@
 
 @bank_accounts_bp.route('/bank_accounts', methods=['GET'])
 def get_bank_accounts():
+    user_id = request.args.get('user_id')
     try:
         cursor, conn = get_db_connection()
         if cursor is None or conn is None:
-            return jsonify({'Error' : 'Failed to connect to server' }), 500
+            logging.debug("Couldn't connect to db.")
```

Comment:
I think this log is redundant here as you probably should have a log within get_db_connection that also puts the actual exception as part of it.